content/docs/1.2.3/deploy/install/_index.md

```diff
@@ -86,8 +86,41 @@ If MountPropagation is disabled, Base Image feature will be disabled.
 
 ### Root and Privileged Permission
 
-The longhorn manager uses `Bidirectional` mount propagation to access the host's disk (default `/var/lib/longhorn`) in the container.
-An use case is to access disk metadata and replicas directory on the host.
+The longhorn components need to access the host's disk in the container.
+A use case is to access disk metadata and replicas directory on the host. The list of the directories of all components are:
+
+- Longhorn Manager
+  - /dev: Volume block device
```

Comment:
Suggest having an explanatory brief statement to say what we need to access `/dev` instead of just a noun. The same as others.